docs/api/actions-and-connectors/create.asciidoc

```diff
@@ -64,8 +63,161 @@ For more information, refer to
 `projectKey`::
 (Required, string) The {jira} project key.
 
-For more information, refer to
-{kibana-ref}/jira-action-type.html[{jira} connector and action].
+For more information, refer to <<jira-action-type>>.
+====
+
+.{webhook-cm} connectors
+[%collapsible%open]
+====
+
+`createCommentJson`::
+(Optional, string) A JSON payload sent to the create comment URL to create a
+case comment. You can use variables to add Kibana Cases data to the payload. The
+required variable is `case.comment`. For example:
++
+[source,json]
+----
+{
+  "body": {{{case.comment}}}
+}
+----
++
+NOTE: Due to Mustache template variables (the text enclosed in triple braces,
+for example, `{{{case.title}}}`), the JSON is not validated when you create the
+connector. The JSON is validated once the Mustache variables have been placed
+when the REST method runs. Manually ensure that the JSON is valid,
+disregarding the Mustache variables, so the later validation will pass.
+
+`createCommentMethod`::
+(Optional, string) The REST API HTTP request method to create a case comment in
+the third-party system. Valid values are either `patch`, `post`, and `put`. The
+default value is `put`.
+
+`createCommentUrl`::
+(Optional, string) The REST API URL to create a case comment by ID in the
+third-party system. You can use a variable to add the external system ID to the
+URL. If you are using the `xpack.actions.allowedHosts` setting, make sure the
+hostname is added to the allowed hosts. For example:
++
+[source,text]
+----
+https://testing-jira.atlassian.net/rest/api/2/issue/{{{external.system.id}}}/comment
+----
+
+`createIncidentJson`::
+(TBD, string) A JSON payload sent to the create case URL to create a case. You
```

Comment:
> (TBD, string) A JSON payload sent to the create case URL to create a case. You

I have these as TBD at this point, since I haven't tested whether they're required or optional.